tests/testJointcal_hsc.py

```diff
@@ -0,0 +1,72 @@
+# See COPYRIGHT file at the top of the source tree.
+
+from __future__ import division, absolute_import, print_function
+
+import unittest
+import os
+
+from astropy import units as u
+
+import lsst.afw.geom
+import lsst.afw.coord as afwCoord
+import lsst.utils
+import lsst.pex.exceptions
+
+import jointcalTestBase
+
+try:
+    data_dir = lsst.utils.getPackageDir('validation_data_hsc')
+    os.environ['ASTROMETRY_NET_DATA_DIR'] = os.path.join(data_dir, 'sdss-dr9-fink-v5b')
+except lsst.pex.exceptions.NotFoundError:
+    data_dir = None
+
+# We don't want the absolute astrometry to become significantly worse
+# than the single-epoch astrometry (about 0.040").
+# This value was empirically determined from the first run of jointcal on
+# this data, and will likely vary from survey to survey.
+absolute_error = 52e-3*u.arcsecond
```

Comment:
Is there a reason you use units.arcsecond here, but lsst.afw.geom.arcseconds later? Please use one or the other throughout if possible.